### PR TITLE
Update user map for Aug 2017 #8056

### LINKED ISSUE #8056: Update user map for Aug 2017
Similar to #7424 

Institutions to add:

Australia | Monash University
-- | --
Australia | Queensland University of Technology
Australia | University of New South Wales
Australia | University of Melbourne
Brazil | IFRJ
Brazil | Universidade Catolica de Brasilia
Brazil | University of Brasilia
Canada | Cégep du Vieux-Montréal
Canada | Centennial College
Canada | Centennial College
Canada | Fleming College
Canada | Ryerson University
Colombia | Universidad del Norte
Finland | Pihlajamäen ala-asteen koulu
Germany | Hertie School of Governance
Germany | Philipps-Universität Marburg
Germany | Stuttgart Media University
Malaysia | HELP University
Malaysia | Universiti Tunku Abdul Rahman
Malaysia | University of Malaya
Mauritius | Afrian Leadership University
Mexico | UAS
Netherlands | Eindhoven University of Technology
Netherlands | Leiden University College
New Zealand | Eastern Institute of Technology
New Zealand | Massey University
New Zealand | University of Auckland
New Zealand | University of Auckland
New Zealand | University of Canterbury
Norway | Høgskolen i Oslo og Akershus
Philippines | De la Salle University
Philippines | Miriam College
Singapore | CWSS
Singapore | Nanyang Polytechnic
Singapore | Ngee Ann Polytechnic
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Singapore | SUTD
South Africa | University of Johannesburg
Sweden | Jönköping International Business School
Thailand | Thammasat University
Turkey | Credemy
UK | City of London School for Girls
UK | University College London
UK | University of Bradford
UK | University of Portsmouth
USA | AGS
USA | Arizona State University
USA | Bettendorf High School
USA | Bryant University
USA | C/3-13IN Basic Training
USA | CCSD59
USA | Central Oregon Community College
USA | Chaffey College
USA | CSU Sacramento
USA | Elon University
USA | Florida International University
USA | George Washington University
USA | Georgia Tech
USA | Georgia Tech
USA | Harvey Mudd College
USA | Indiana State University
USA | Indiana State University
USA | Jackson Heights Middle School
USA | Medical College of Wisconsin
USA | Medical College of Wisconsin
USA | Miami University
USA | Miami University
USA | Michigan State University
USA | Michigan State University
USA | Mississippi State University
USA | Montclair High School
USA | Nevada State College
USA | New England Insitute of Technology
USA | New Mexico State Univeristy
USA | North Carolina State University
USA | Northeastern University
USA | Northern Arizona University
USA | Northwestern College
USA | PACE
USA | Pitzer College
USA | Presbyterian Christian School
USA | San Diego State University
USA | Seattle Academy of Arts and Sciences
USA | Southeast Community College
USA | Southern Illinois University Edwardsville
USA | Spring Hill College
USA | University at Albany
USA | University at Albany, SUNY
USA | University if Minnesota
USA | University of Alabama at Birmingham
USA | University of California Berkeley
USA | University of Dallas
USA | University of Florida
USA | University of Kentucky
USA | University of Maryland, College Park
USA | University of Massachusetts Boston
USA | University of Minnesota
USA | University of North Carolina at Charlotte
USA | University of South Dakota
USA | University of Texas at Arlington
USA | University of Texas at Austin
USA | University of the Incarnate Word
USA | University of Washington
USA | University of West Florida
USA | West Chester University
USA | Western Carolina University
USA | Western Kentucky University
USA | William Penn University
USA | Worcester State University

### PR DESCRIPTION
**Pull request by g3chenyigmailcom**
added new institutions to userMapData.json

Fixes #8056

### CODE REVIEW

**Comment by g3chenyigmailcom** [comment]
Can someone review this? Thanks.

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -141,10 +141,12 @@
       "Unicesumar",
       "Unificados AEDB 2016",
       "UniFOA",
+      "Universidade Catolica de Brasilia",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Entry already exists, one line below.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -511,6 +519,7 @@
     ],
     "Malaysia": [
       "Help University",
+      "HELP University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Entry already exists one line above.

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -539,24 +548,26 @@
       "University of Reading"
     ],
     "Mauritius": [
+      "Afrian Leadership University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Looks like a typo of the existing entry one line below.

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -665,11 +678,13 @@
       "Corazon Locsin Montelibano Memorial Regional Hospital",
       "De La Salle College of Medicine",
       "De La Salle Health Sciences Institute",
+      "De la Salle University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Entry already exists one line below.

File: src/main/webapp/data/userMapData.json [hunk 5]
```diff
@@ -770,6 +786,7 @@
       "St. Gabriels' Secondary",
       "St. Joseph's Institution",
       "St. Margaret's Secondary School",
+      "SUTD",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This stands for `Singapore University of Technology and Design`, which is an existing entry above.

File: src/main/webapp/data/userMapData.json [hunk 6]
```diff
@@ -829,6 +847,7 @@
     ],
     "Sweden": [
       "International School of the Stockholm Region",
+      "Jönköping International Business School",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This is actually part of `Jönköping University` which is an existing entry one line below.

File: src/main/webapp/data/userMapData.json [hunk 7]
```diff
@@ -1620,9 +1666,10 @@
       "University of New Hampshire",
       "University of New Mexico",
       "University of North Carolina",
-      "University of North Carolina Greensboro",
-      "University of North Carolina at Chapel Hill",
       "University of North Carolina - Wilmington",
+      "University of North Carolina at Chapel Hill",
+      "University of North Carolina at Charlotte",
+      "University of North Carolina Greensboro",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Can add an `at` here too for consistency.

File: src/main/webapp/data/userMapData.json [hunk 8]
```diff
@@ -1558,19 +1601,21 @@
...
       "University of Alabama at Birmingham",
       "University of Alaska",
       "University of Alaska Anchorage",
       "University of Alaska-Fairbanks",
-      "University at Albany",
       "University of Arizona",
       "University of Arizona South",
       "University of Arkansas",
       "University of California",
+      "University of California Berkeley",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Already exists 3 lines below.

File: src/main/webapp/data/userMapData.json [hunk 9]
```diff
@@ -1558,19 +1601,21 @@
       "Union Mine High School",
       "United States Military Academy Preparatory School",
       "Univeristy of North Georgia",
+      "University at Albany",
       "University at Albany, SUNY",
       "University at Buffalo",
+      "University if Minnesota",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Typo of an entry you already added.

File: src/main/webapp/data/userMapData.json [hunk 10]
```diff
@@ -1558,19 +1601,21 @@
       "Union Mine High School",
       "United States Military Academy Preparatory School",
       "Univeristy of North Georgia",
+      "University at Albany",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Same as the entry one line below.

File: src/main/webapp/data/userMapData.json [hunk 11]
```diff
@@ -1355,10 +1389,13 @@
       "Navarro College",
       "NC A&T State University",
       "NC State University",
+      "Nevada State College",
       "New Bern High School",
       "New Brunswick Health Sciences Technology High School",
+      "New England Insitute of Technology",
       "New Fairfield High School",
       "New Jersey Institute of Technology",
+      "New Mexico State Univeristy",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Duplicate of entry one line below.

File: src/main/webapp/data/userMapData.json [hunk 12]
```diff
@@ -1399,12 +1437,13 @@
       "Old Dominion University",
       "Olivia Nieto-Herrera Elementary",
       "Omada Health",
-      "On-Ramps",
       "oneida-boces.org",
+      "On-Ramps",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: The original ordering was correct.

File: src/main/webapp/data/userMapData.json [hunk 13]
```diff
@@ -1189,6 +1219,7 @@
       "Georgia Gwinnett College",
       "Georgia Institute of Technology",
       "Georgia State University",
+      "Georgia Tech",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Same as `Georgia Institute of Technology` two lines above.

File: src/main/webapp/data/userMapData.json [hunk 14]
```diff
@@ -1130,6 +1157,8 @@
       "Dudley High School",
       "Duke University",
       "Duluth Middle School",
+      "D'Youville College",
+      "D'Youville College School of Pharmacy",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Can remove since it's part of `D'Youville College`.

File: src/main/webapp/data/userMapData.json [hunk 15]
```diff
@@ -1104,13 +1132,12 @@
       "Creighton University",
       "Crooms Academy of Information Technology",
       "CSU Chico",
+      "CSU Sacramento",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Use the full name `California State University, Sacramento`.

File: src/main/webapp/data/userMapData.json [hunk 16]
```diff
@@ -1104,13 +1132,12 @@
       "Creighton University",
       "Crooms Academy of Information Technology",
       "CSU Chico",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Not your fault, but can help remove this entry? It's a duplicate of the existing entry `California State University, Chico` above.

File: src/main/webapp/data/userMapData.json [hunk 17]
```diff
@@ -1057,18 +1082,21 @@
       "Carthage College",
       "Catawba Valley Community College",
       "Cauldron Ice Cream, Inc.",
+      "CCSD59",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Use the full name `Community Consolidated School District 59`.

**Comment by g3chenyigmailcom** [comment]
seems like still need a human eye to double check...:)

File: src/main/webapp/data/userMapData.json [hunk 18]
```diff
@@ -1097,20 +1119,18 @@
       "Columbus College of Art and Design",
       "Columbus State Community College",
       "Community Partnership Charter School",
+      "Community Consolidated School District 59",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Wrong order?

**Comment by whipermr5** [comment]
@g3chenyigmailcom Thanks for your contribution! May we have your full name for inclusion into the contributors' list? :)

**Comment by g3chenyigmailcom** [comment]
No problem! My name is Yi Chen, you can also call me Felix :)

**Pull request merged**